### PR TITLE
NR-155179 cloud quickstart dashboards

### PR DESCRIPTION
**Pull request by rajut-xrg**
# Summary
Added new quickstart dashboards
1. GCP Cloud Run
2. GCP Cloud SQL
3. GCP Dataproc
4. GCP Application Load Balancer
5. GCP Redis
6. GCP VPC Access Connector

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [N/A] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [N/A ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

**Comment by rajut-xrg** [comment]
Hi @seemantk ,

File: dashboards/gcp-cloud-run/gcp-cloud-run.json [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            "yAxisLeft": {
+              "zero": true
+            },
+            "yAxisRight": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Memory allocation time(Gigabytes-Seconds)",
```
Comment (d3caf) [inline dashboards/gcp-cloud-run/gcp-cloud-run.json:131]: Can we put a space between the title and the parenthesis?

```suggestion
"Memory allocation time (gigabytes-seconds)"
```

Comment (rajut-xrg) [reply dashboards/gcp-cloud-run/gcp-cloud-run.json:131]: Hey @d3caf , Is it time/title?

File: dashboards/gcp-cloud-run/gcp-cloud-run.json [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT SUM(`gcp.run.request_count`) AS 'Total Requests', SUM(`gcp.run.container.memory.allocation_time`) AS 'Total Memory Allocation Time(Gigabytes-Seconds)', SUM(`gcp.run.container.billable_instance_time`) AS 'Total Billable Instance Time(Seconds)', SUM(`gcp.run.container.cpu.allocation_time`) AS 'Total CPU Allocation Time(Seconds)'"
```
Comment (d3caf) [inline dashboards/gcp-cloud-run/gcp-cloud-run.json:28]:
```suggestion
                "query": "FROM Metric SELECT SUM(`gcp.run.request_count`) AS 'Total Requests', SUM(`gcp.run.container.memory.allocation_time`) AS 'Total Memory Allocation Time (gigabytes-seconds)', SUM(`gcp.run.container.billable_instance_time`) AS 'Total Billable Instance Time (seconds)', SUM(`gcp.run.container.cpu.allocation_time`) AS 'Total CPU Allocation Time (seconds)'"
```

File: dashboards/gcp-cloud-run/gcp-cloud-run.json (hunk 1 above)
Comment (d3caf) [reply dashboards/gcp-cloud-run/gcp-cloud-run.json:131]: @rajut-xrg I was just thinking it'd be good to add a space between the title and parentheses and also probably don't need to capitalize the words in there.

File: dashboards/gcp-data-proc/gcp-data-proc.json [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+            "yAxisLeft": {
+              "zero": true
+            },
+            "yAxisRight": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Memory size by status(GB)",
```
Comment (d3caf) [inline dashboards/gcp-data-proc/gcp-data-proc.json:209]:
```suggestion
          "title": "Memory size by status (GB)",
```

**Review by d3caf — COMMENTED** [review]
Just a few minor content adjustments, otherwise looks great! Approving since they're not blocking notes.

**Review by d3caf — APPROVED** [review]

**Comment by rajut-xrg** [comment]
Hey @d3caf , added more dashboards could you please review the changes. or add reviews who can.
FYI.. There are some more to come. As the changes are many so splitting the commits.

**Pull request closed without merging**